### PR TITLE
Refactor makefile to separate use cases

### PR DESCRIPTION
**Pull request by sudo-bmitch**
This builds on top of #318 and it should be merged first.

This splits the k8s install, dev environment, FRSCA components, and examples.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -205,12 +190,12 @@ lint-spellcheck:
 lint-yaml: ## Lint yaml files
 	yamllint .
 
-.PHONY: fmt-md ## Format markdown files
-fmt-md:
+.PHONY: fmt-md
+fmt-md: ## Format markdown files
 	npx --yes prettier --write --prose-wrap always **/*.md
```
Comment (rgreinho) [inline Makefile:195]: I had made a mistake here and I forgot the quotes 🤦 Maybe we can fix it at the same time?

```suggestion
	npx --yes prettier --write --prose-wrap always "**/*.md"
```

**Review by rgreinho — DISMISSED** [review]
👍

File: README.md [hunk 2]
```diff
@@ -48,21 +48,25 @@ pipeline run:
...
-   6. [Vault]
-   7. [Kyverno]
+   1. [registry]
+   1. [SPIFFE/Spire]
+   1. [Vault]
+1. Install and setup FRSCA's components which include:
+   1. [Tekton Pipelines]
+   1. [Tekton Chains]
+   1. [Kyverno]
+1. Setup a mirror of example repositories and tekton triggers for each mirror.
```
Comment (bradbeck) [inline README.md:69]: Seems a bit odd to have `1.` for all of these items.

Comment (sudo-bmitch) [reply README.md:69]: It's how markdown does it if you want the items to be automatically numbered. It allows you to add/remove items from the middle of the list without renumbering.

**Review by rgreinho — APPROVED** [review]

**Review by bradbeck — APPROVED** [review]

**Pull request merged**